pkg/cmd/dashlanetemplatefuncs.go

```diff
@@ -33,14 +54,13 @@ func (c *Config) dashlanePasswordTemplateFunc(filter string) any {
 		panic(err)
 	}
 
-	c.Dashlane.cache[filter] = data
+	c.Dashlane.cachePassword[filter] = data
 	return data
 }
 
 func (c *Config) dashlaneOutput(args ...string) ([]byte, error) {
 	name := c.Dashlane.Command
 	cmd := exec.Command(name, append(c.Dashlane.Args, args...)...)
-	cmd.Stdin = os.Stdin
```

Comment:
`dcli note` will ask the user to select a note if multiple notes are matching the filter.
Removing the access to `stdin` leads to the command line exiting on error right away when this happens.

This however does not allow to enter the master password for Dashlane if needed - that being said, those are being asked on stdout at the moment, which is not visible to the user and can only lead to confusion.

Reply:
Thanks, makes sense.